src/components/Model/NewForm.js

```diff
@@ -98,23 +129,23 @@ class NewForm extends Component {
 
 const mapStateToProps = state => {
     return {
+        engine: state.sideNav.engine,
```

Comment:
Why is this info stored in sideNav?

Reply:
No idea, I think Nic put the engines redux stuff in the sidenav reducer. Would be clearer if there was an engineReducer or appReducer. it works for now, I have a reminder to clean it up later.